parsers/cobol.c

```diff
@@ -132,21 +227,37 @@ static void initializeCobolParser (langType language)
 	cobol = language;
 
 	addLanguageCallbackRegex (cobol,
-					  COBOL_REGEX_PREFIX
-					  "[0-9]+[ \t]+([A-Z0-9][A-Z0-9-]*)[ \t]+("
-					  "BLANK|OCCURS|IS|JUST|PIC|REDEFINES|RENAMES|SIGN|SYNC|USAGE|VALUE"
-					  ")",
-					  "{icase}",
-					  make_tag_for_data_maybe, NULL, &cobol);
+		COBOL_REGEX_PREFIX_FIXED
+		"[0-9]+[ \t]+([A-Z0-9][A-Z0-9_-]*)[ \t]+("
+		"BLANK|OCCURS|IS|JUST|PIC|REDEFINES|RENAMES|SIGN|SYNC|USAGE|VALUE|COMP\\-1|COMP\\-2|COMP\\-3|COMP\\-X|COMP"
+		")",
+		"{icase}",
+		make_tag_for_data_maybe, NULL, &cobol);
+
+	addLanguageCallbackRegex(cobol,
+		COBOL_REGEX_PREFIX_RELAXED
+		"[0-9]+[ \t]+([A-Z0-9][A-Z0-9_-]*)[ \t]+("
+		"BLANK|OCCURS|IS|JUST|PIC|REDEFINES|RENAMES|SIGN|SYNC|USAGE|VALUE|COMP\\-1|COMP\\-2|COMP\\-3|COMP\\-X|COMP"
+		")",
+		"{icase}",
+		make_tag_for_data_maybe, NULL, &cobol);
+
 	addLanguageCallbackRegex (cobol,
-					  COBOL_REGEX_PREFIX
-					  "([A-Z0-9][A-Z0-9-]*)\\.",
-					  "{icase}",
-					  make_tag_for_paragraph_maybe, NULL, &cobol);
+		COBOL_REGEX_PREFIX_FIXED
```

Comment:
Can we use COBOL_REGEX_PREFIX_UNIFIED here.
So you can uinfy two addLanguageCallbackRegex into one.